packages/backend-storage/src/construct.ts

```diff
@@ -46,6 +46,26 @@ export class AmplifyStorage
 
     const bucketProps: BucketProps = {
       versioned: props.versioned || false,
+      cors: [
+        {
+          maxAge: 3000,
+          exposedHeaders: [
+            'x-amz-server-side-encryption',
+            'x-amz-request-id',
+            'x-amz-id-2',
+            'ETag',
+          ],
+          allowedHeaders: ['*'],
+          allowedOrigins: ['*'],
+          allowedMethods: [
+            HttpMethods.GET,
+            HttpMethods.HEAD,
+            HttpMethods.PUT,
+            HttpMethods.POST,
+            HttpMethods.DELETE,
+          ],
+        },
+      ],
```

Comment:
should this be behind a `enableCors` prop so we don't always enable CORS for everyone whether they want it or not.

Reply:
My thought here was convention > configuration. In Gen1 we don't expose this as a prop and product hasn't asked us to. It's easier to add config options than take them away

Reply:
Discussed offline, the risk is minimal, if any, and will be included in the threat model.